### ISSUE TITLE
Parser errors

### ISSUE THREAD

**Issue by burtbeckwith** [openrewrite/rewrite-micronaut#14]
I'm testing the Micronaut module with https://github.com/oracle-quickstart/oci-micronaut. When running in src/api, I get some error output:

```
line 1:68 token recognition error at: '<'
line 1:68 token recognition error at: '<'
line 1:69 token recognition error at: '?'
line 1:69 token recognition error at: '?'
line 1:70 mismatched input ' ' expecting {')', '[', ',', '.', '*', '&&', '||', '..', Identifier}
line 1:70 mismatched input ' ' expecting {')', '[', ',', '.', '*', '&&', '||', '..', Identifier}
line 1:109 token recognition error at: '>'
line 1:109 token recognition error at: '>'
```

The only source that seems to have `<?` followed by a space is in src/api/src/main/java/api/services/support/TrackEventFilter.java:

`private Flowable<? extends MutableHttpResponse<? extends Object>> trackEvent(`

When running in src/carts (which uses a Maven build) I get the same messages but not duplicated:

```
line 1:68 token recognition error at: '<'
line 1:69 token recognition error at: '?'
line 1:70 mismatched input ' ' expecting {')', '[', ',', '.', '*', '&&', '||', '..', Identifier}
line 1:109 token recognition error at: '>'
```

but I don't see any source that might have triggered this in that module.

I'm using v5.4.1 of the Gradle plugin, v0.2.0 of rewrite-micronaut, and `activeRecipe("org.openrewrite.java.micronaut.Micronaut2to3Migration")`

**Comment by pway99** [comment]
Hello @burtbeckwith, Are you running the latest release or snapshot? These look like antlr grammar issues, which should have been fixed by https://github.com/openrewrite/rewrite-micronaut/commit/27adddf7477e431b9a0f86bb81775c8f2f91ed26.

**Comment by burtbeckwith** [comment]
Yep, fixed in 0.3.0-SNAPSHOT
